Game.c

```diff
@@ -124,12 +148,29 @@ int solve(gameData * game, char * path) {
 
 int edit(gameData * game, char* path,){
 	if (path == NULL) {
-
+		printf(ERROR_INV_CMD);
 	}
-}
 
-int mark_errors(gameData * game){
+}
 
+int mark_errors(gameData * game, char ** cmdArr){
+	int i;
+	if (!checkInt(cmdArr[0])) {/*check if valid int*/
```

Comment:
first check if game->mode == 1, if not print inv cmd.
if it is then check the argument. because parser already checks if the argument exists.

Reply:
added